diff --git a/app/build.gradle b/app/build.gradle
index 3eee686..af77417 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -5,8 +5,8 @@ plugins {
 
 apply from: rootProject.file('flavors.gradle')
-apply from: rootProject.file('signings.gradle')
-apply from: rootProject.file('buildRelease.gradle')
+//apply from: rootProject.file('signings.gradle')
+//apply from: rootProject.file('buildRelease.gradle')
 
 base {
     archivesName = "siyuan-${siyuanVersionName}"
@@ -46,6 +46,23 @@ android {
         versionName siyuanVersionName
     }
 
+    signingConfigs {
+        release {
+            def ksFile = file("${rootDir}/keystore/siyuan.jks")
+            if (ksFile.exists()) {
+                storeFile ksFile
+                storePassword "123456"
+                keyAlias "siyuan"
+                keyPassword "123456"
+            } else {
+                storeFile ksFile
+                storePassword "123456"
+                keyAlias "siyuan"
+                keyPassword "123456"
+            }
+        }
+    }
+
     buildFeatures {
         buildConfig true
     }
@@ -53,7 +70,7 @@ android {
     buildTypes {
         release {
-            signingConfig signingConfigs.siyuanConfig
+            signingConfig signingConfigs.release
 
             minifyEnabled true
             shrinkResources true
@@ -106,7 +123,7 @@ dependencies {
     implementation 'commons-io:commons-io:2.21.0'
     implementation 'com.gitee.zackratos:UltimateBarX:0.8.1'
     implementation 'com.blankj:utilcodex:1.31.1'
     implementation 'com.koushikdutta.async:androidasync:3.1.0'
-    implementation(name: 'kernel', ext: 'aar')
+    implementation files('libs/kernel.aar')
 }
