diff --git a/kernel/api/system.go b/kernel/api/system.go
index 9075ef06a2717b3965c8f932ddf1b9b02a9d66ff..469d2d08378984b32474681d778014a297d4295c 100644
--- a/kernel/api/system.go
+++ b/kernel/api/system.go
@@ -421,9 +421,6 @@ func normalizeCustomEmojiPath(name, ext string) (string, error) {
 
 var checkUpdate = contractHandler(apicontract.SystemCheckUpdate, func(c *gin.Context, request apicontract.SystemCheckUpdateRequest) (ret apicontract.Response[apicontract.Null]) {
 	ret = apicontract.Success(apicontract.Null{})
-
-	showMsg := request.ShowMsg
-	model.CheckUpdate(showMsg)
 	return
 })
 
diff --git a/kernel/conf/system.go b/kernel/conf/system.go
index 51ae3802eea81f3e7e297acc743f893e9b491170..429eeba8471a831513957041662e0c4b27edefa0 100644
--- a/kernel/conf/system.go
+++ b/kernel/conf/system.go
@@ -79,7 +79,7 @@ func NewSystem() *System {
 		Name:               util.GetDeviceName(),
 		KernelVersion:      util.Ver,
 		NetworkProxy:       &NetworkProxy{},
-		DownloadInstallPkg: true,
+		DownloadInstallPkg: false,
 		UpdateChannel:      UpdateChannelStable,
 	}
 }
diff --git a/kernel/model/mount.go b/kernel/model/mount.go
index 6552d1315259de8b4b3e9e8a45e629db1bedea9b..ff5517a5c6a555f169cb592a358d8e13ab993771 100644
--- a/kernel/model/mount.go
+++ b/kernel/model/mount.go
@@ -528,8 +528,8 @@ func mountBox(boxID string) (alreadyMount bool, err error) {
 		task.AppendAsyncTaskWithDelay(task.PushMsg, 3*time.Second, util.PushErrMsg, Conf.Language(244), 7000)
 		go func() {
 			// 每次打开帮助文档时自动检查版本更新并提醒 https://github.com/siyuan-note/siyuan/issues/5057
-			time.Sleep(time.Second * 10)
-			CheckUpdate(true)
+			// time.Sleep(time.Second * 10)
+			// CheckUpdate(true)
 		}()
 	}
 
diff --git a/kernel/model/updater.go b/kernel/model/updater.go
index 8a39a626b1e2b410457b7ef1bd23590462320fbc..6db50b9667fddfdf56b18d1b02bbe19dbd8566aa 100644
--- a/kernel/model/updater.go
+++ b/kernel/model/updater.go
@@ -267,6 +267,10 @@ func isVersionUpToDate(releaseVer string) bool {
 var skipInstallPkgPlatformCached = -1
 
 func skipNewVerInstallPkg() bool {
+	// Conf.System.DownloadInstallPkg(自动下载更新安装包) 强制赋值为false
+	Conf.System.DownloadInstallPkg = false
+	Conf.Save()
+
 	if skipInstallPkgPlatformCached == -1 {
 		skipInstallPkgPlatformCached = 0
 		if !gulu.OS.IsWindows() && !gulu.OS.IsDarwin() {
